### ISSUE TITLE
Diagram shows only after restart

### ISSUE THREAD

**Issue by rustam-python** [zsviczian/excalibrain#25]
Hello [zsviczian](https://github.com/zsviczian)!

Please advise:
Why excalibrain shows diagram only after I restart Obsidian app?

Obsidian shows me this picture:

![image](https://user-images.githubusercontent.com/41113512/167584457-d1d26b85-10a1-4a04-abb4-b885ebc695d2.png)

and ony after that I can see the diagram.

Should I enable something in DataView?

I use:
- Windows 10 21H2; 
- Obsidian 0.14.6;
- ExcaliBrain – 0.0.9;
- DataView – 0.4.26;
- excaliDraw – 1.6.28.

Thank you for this AWESOME plugin!

**Comment by rustam-python** [comment]
But after DV index update ExcaliBrain should generate the graph layout for new notes? Or I need to wait next DV index update?

There are 3650 notes in my vault for now.

**Comment by zsviczian** [comment]
yes. there was an issue with new page updates on v0.0.8, but that should work ok in 0.0.9
